RaveBusinessLogic/BRAT.xml

```diff
@@ -11,11 +11,6 @@
           <Node xpathlabel="Name" xpath="HTMLFile" type="report" />
           <Node label="Capacity">
             <Children>
-              <Node label="Existing Dam Capacity" xpath="Vector" type="line" symbology="oCC_EX" id="ex_capacity" />
-              <Node label="Existing Dam Size" xpath="Vector" type="line" symbology="mCC_EX_CT" />
-              <Node label="Historic Dam Capacity" xpath="Vector" type="line" symbology="oCC_HPE" id="his_capacity" />
-              <Node label="Historic Dam Size" xpath="Vector" type="line" symbology="mCC_HPE_CT" />

```

Comment:
Is this just getting rid of because it is redundant now with pyBRAT? Maybe ask @MattReimer or @philipbaileynar to weigh in here. Leaving these does not break anything and just means it will work if you're dealing with shapefiles. Since we are dealing with geopackages, this shouldn't be the case.